romtools/vector_space/utils/truncater.py

```diff
@@ -65,7 +65,7 @@ class LeftSingularVectorTruncater(Protocol):
     Interface for the Truncater class.
     '''
 
-    def truncate(self, basis: np.ndarray,  singular_values: np.ndarray) -> np.ndarray:
+    def truncate(self, basis: np.ndarray, singular_values: np.ndarray) -> np.ndarray:
```

Comment:
The `singular_values` argument is only used by `EnergyBasedTruncater`. 

I've disabled the `unused-argument` warnings on the other classes for now, but I wonder if there's a better solution.